knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml

```diff
@@ -728,7 +728,7 @@ metadata:
 spec:
   # this deployment is going to be scaled up by the
   # controller when the very first KafkaChannel is created
```

Comment:
Should we remove this comment since default is 1?

Reply:
yeah, we can patch remove the comments too